package.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"types": "./lib/index.d.ts",
 	"dependencies": {
-		"ardrive-core-js": "1.0.4",
+		"ardrive-core-js": "../ardrive-core-js_PE-464_download_file",
```

Comment:
TODO: Update

Reply:
That core branch is the right one

Reply:
Thanks, but the reminder is just to update this back to the correct npm release before merging :)